db/patch68.sql

```diff
@@ -3,4 +3,4 @@
 -- This resolves https://joindin.jira.com/browse/JOINDIN-725
 ALTER DATABASE joindin CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-INSERT INTO patch_history SET patch_number = 68;
\ No newline at end of file
+INSERT INTO patch_history SET patch_number = 69;
```

Comment:
Would argue that these should just be ticket-ref.sql and the person who merges renames and adds this line. Asking for conflicts the way its currently set up